### ISSUE TITLE
Update SSB's SSP to cover recent changes

### ISSUE THREAD

**Issue by mogul** [GSA/data.gov#3748]
## User Story

In order to satisfy assessment and authorization needs, data.gov's GSA ISSM wants the SSB SSP to accurately reflect recent changes to the EKS broker.

## Acceptance Criteria

[ACs should be clearly demoable/verifiable whenever possible. Try specifying them using [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications).]

- [ ] GIVEN I look at [the most recent version of the SSB SSPP](https://docs.google.com/document/d/1SzqVjIDowxtfjWljXDUM8xzO8zzoQPq1/edit?rtpof=true) \
  WHEN I look for any [changes that we have implemented recently](https://github.com/GSA/data.gov/issues/3577) \
  THEN I find that they are all covered in the SSPP
- [ ] WHEN I look at [the most recent version of the data.gov SSPP](https://docs.google.com/document/d/11le1yfXjGlk6pOpXryVH4TSJNsSBgp9X/edit) \
  THEN I find that our implementation of cg-egress-proxy is documented  in SC-7
  AND our system diagrams in section 10 includes cg-egress-proxy and the public_egress spaces \
  AND our diagram and description of the backup-manager data flow is accurate.

## Background

For reference, this is the companion [data.gov SSPP](https://docs.google.com/document/d/11le1yfXjGlk6pOpXryVH4TSJNsSBgp9X/edit)

## Security Considerations ([required](https://nvd.nist.gov/800-53/Rev4/control/CM-4))

[comment]: # "Our SSP says 'The Data.gov team ensures security implications are considered as part of the agile requirements refinement process by including a section in the issue template used as a basis for new work.' so please don't remove this section without care."
This documentation update is required by the NIST RMF process. It's done as a separate, individually-tracked effort because we don't yet have a way to do this continuously as part of our technical implementation.

## Sketch

[New "SSPP" form document](https://docs.google.com/document/d/11le1yfXjGlk6pOpXryVH4TSJNsSBgp9X/edit)

Known-needed topics:

- Removal of Fargate, inclusion of Managed Node Groups
  - Anything narrative/control that relied on Fargate describes the equivalent control for MNGs
  - Addition of AWS Inspector, and description of EC2 and ECR scanning
  - Use of GSA-provided AMIs that are CIS-hardened
-  Addition of persistent volumes
- Switching from ALB to NLB for load-balancer
- Egress control for EKS workloads
- Egress control for cloud.gov workloads

**Comment by hkdctol** [comment]
@albionzeglin-gsa this ticket has the links to google docs of the SSPPs

**Comment by hkdctol** [comment]
I have reviewed the front sections of the Data.gov SSPP at https://docs.google.com/document/d/11le1yfXjGlk6pOpXryVH4TSJNsSBgp9X/edit?usp=sharing&ouid=113511966954817069922&rtpof=true&sd=true there are only very minimal changes on this one, so just need someone to look at the few places where I commented to either confirm no changes needed or make the applicable updates re references to Solr. I will take a first pass at the SSB SSPP as well.

**Comment by hkdctol** [comment]
Took a pass at SSB SSPP. Changes needed in front section are minimal. https://docs.google.com/document/d/1SzqVjIDowxtfjWljXDUM8xzO8zzoQPq1/edit?usp=sharing&ouid=113511966954817069922&rtpof=true&sd=true

**Comment by jbrown-xentity** [comment]
@hkdctol reviewed both docs, and all comments are addressed. Please let me know if you have anything additional.

**Comment by hkdctol** [comment]
@albionzeglin-gsa I think what you could do now is given that the problems/remaining issues are described in https://docs.google.com/document/d/1_2GroL3kkzgWePPcNhzRD6SFDc-5Kt0ic8DiZln_TAs/edit?usp=sharing
you can start to look at the control sections of [SSB SSPP](https://docs.google.com/document/d/1SzqVjIDowxtfjWljXDUM8xzO8zzoQPq1/edit?usp=sharing&ouid=113511966954817069922&rtpof=true&sd=true)
and [Data.gov SSPP](https://docs.google.com/document/d/11le1yfXjGlk6pOpXryVH4TSJNsSBgp9X/edit?usp=sharing&ouid=113511966954817069922&rtpof=true&sd=true) and start commenting on which sections may need an update once we've finally settled on Solr questions.

**Comment by hkdctol** [comment]
@albionzeglin-gsa as you're looking at the control sections, can you confirm that the docs are following the current [template](https://docs.google.com/document/d/1OiNfvlo5UtTC4u0kP5cvs7rGrI1Pg7V9/edit?usp=sharing&ouid=113511966954817069922&rtpof=true&sd=true) and if there are any format/template/numbering changes, go ahead and start making those?

**Comment by hkdctol** [comment]
Reviewed again in light of Solr leader-follower, there are 1-2 changes only in the SSB document (front sections not controls)

**Comment by hkdctol** [comment]
just making one more change to diagram, before migration

**Comment by mogul** [comment]
Here's the PR to update the diagram... Once this is approved/merged, I'll copypasta into the Google Doc.
https://github.com/GSA/datagov-compliance/pull/32

**Comment by hkdctol** [comment]
Since we have handled the front section of the SSPP's, will mark this one as done and create a new ticket for the continuing work on controls.

**Comment by hkdctol** [comment]
This was completed
